src/olympia/addons/tests/test_views.py

```diff
@@ -573,7 +573,7 @@ def test_review_microdata_personas(self):
         a.save()
         response = self.client.get(reverse('addons.detail', args=['a15663']))
         html = pq(response.content)('table caption').html()
-        assert '&lt;script&gt;alert(&#34;fff&#34;)&lt;/script&gt;' in html
+        assert '&lt;script&gt;alert("fff")&lt;/script&gt;' in html
```

Comment:
fair point, I'll check. We noticed that the escape difference is that django uses `&quot;` instead of `&#34;` but otherwise it should escape it.